assets/stylesheets/shared/_colors.scss

```diff
@@ -1,74 +1,77 @@
+/**
+ * Colors
+ *
+ * Color variables based on the design handbook
+ * See wordpress.com/design-handbook/colors/ for more info.
+ */
+
 // Primary Accent (Blues)
 $blue-wordpress:         #0087be;
 $blue-light:             #78dcfa;
 $blue-medium:            #00aadc;
 $blue-dark:              #005082;
 
-// Jetpack Green
-$green-jetpack:          #8cc258;
-
 // Grays
 $gray:                   #87a6bc;
 $gray-light:             lighten( $gray, 33% ); //#f3f6f8
+$gray-lighten-30:        lighten( $gray, 30% ); //#e9eff3
+$gray-lighten-20:        lighten( $gray, 20% ); //#c8d7e1
+$gray-lighten-10:        lighten( $gray, 10% ); //#a8bece
+$gray-darken-10:         darken( $gray, 10% ); //#668eaa
+$gray-darken-20:         darken( $gray, 20% ); //#4f748e
+$gray-darken-30:         darken( $gray, 30% ); //#3d596d
 $gray-dark:              darken( $gray, 38% ); //#2e4453
 
 // $gray-text: ideal for standard, non placeholder text
 // $gray-text-min: minimum contrast needed for WCAG 2.0 AA on white background
 $gray-text:              $gray-dark;
 $gray-text-min:          darken( $gray, 18% ); //#537994
 
-// $gray color functions:
-// lighten( $gray, 10% ) //#a8bece
-// lighten( $gray, 20% ) //#c8d7e1
-// lighten( $gray, 30% ) //#e9eff3
-// darken( $gray, 10% ) //#668eaa
-// darken( $gray, 20% ) //#4f748e
-// darken( $gray, 30% ) //#3d596d
-//
-// See wordpress.com/design-handbook/colors/ for more info.
-
 // Secondary Accent (Oranges)
-$orange-jazzy:           #f0821e;
 $orange-fire:            #d54e21;
+$orange-jazzy:           #f0821e;
 
-// Alerts
-$alert-yellow:           #f0b849;
-$alert-red:              #d94f4f;
-$alert-green:            #4ab866;
-$alert-purple:           #855DA6;
-
-// Link hovers
-$link-highlight:         tint($blue-medium, 20%);
+// Validations and Alerts
+$green-valid:            #4ab866;
+$yellow-warning:         #f0b849;
+$red-error:              #d94f4f;
+$alert-yellow:           $yellow-warning;
+$alert-red:              $red-error;
+$alert-green:            $green-valid;
```

Comment:
The colors however _are_ called `$alert-red` in the Handbook not `$red-error` – and it's currently consistently used on web, iOS and Android. From the Handbook:

![screen shot 2017-07-19 at 10 41 58](https://user-images.githubusercontent.com/4389/28360946-eeec3544-6c6e-11e7-8fe6-9a4560ead8d0.png)

Let's not introduce new variables and new indirections, nor use a PR to do two changes in one — PRs should be as much as sensibly relevant doing just one thing, otherwise it's hard to discuss, review, and find the decisions later.